src/Format/Ini/Value.cpp

```diff
@@ -76,7 +76,7 @@ int Value::toInt() const
     {
         return std::stoi(_value);
     }
-    catch (std::invalid_argument)
```

Comment:
I'm interested, why exceptions often handled by non-const reference?
Will it be changed? =)